### PR TITLE
BETA: Register thikhathali.is-a.dev

### PR DESCRIPTION
**Pull request by Thikhathali**
Added `thikhathali.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/thikhathali.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Thikhathali",
+        "email": "thikhamataboge@gmail.com"
+    },
+    "record": {
+        "CNAME": "thikhathali.github.io"
```
Comment (CuteDog5695) [inline domains/thikhathali.json:7]: Cant access the website (says that page doesnt exist)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/thikhathali.json (hunk 1 above)
Comment (CuteDog5695) [reply domains/thikhathali.json:7]: Rename the github repo that your website is in to thikhathali.github.io

Comment (Thikhathali) [reply domains/thikhathali.json:7]: Just did.

**Review by iostpa — APPROVED** [review]
Fine now since website is accessible now

**Comment by Thikhathali** [comment]
Thank you.

**Pull request merged**